orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Execution.java

```diff
@@ -196,13 +197,14 @@ public void setEndTime(@Nullable Long endTime) {
    * Gets the start ttl timestamp for this execution. If the execution has not
    * started before this timestamp, the execution will immediately terminate.
    */
-  private Long startTimeTtl;
+  private Instant startTimeTtl;
 
-  public @Nullable Long getStartTimeTtl() {
+  public @Nullable
+  Instant getStartTimeTtl() {
```

Comment:
Not to be picky but is this really a TTL?

Reply:
What would you call it instead? It's time-to-live as far as it being valid. It's not a cache, but it becomes invalid after that timestamp.

Reply:
I’m not bringing you solutions, I’m just bringing you problems.